assets/js/components/display.js

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable arrow-body-style */
+const display = (() => {
+  // eslint-disable-next-line arrow-body-style
+  const createTable = () => {
+    return `<div class="status text-center">
```

Comment:
You should write all these `HTML` codes in your `index.html` file and do your DOM manipulation in a different file say `domManager.js`

Reply:
Move all HTML related code from this place.